airgun/views/os.py

```diff
@@ -9,6 +9,7 @@
 class OperatingSystemView(BaseLoggedInView):
     title = Text("//h1[text()='Operating systems']")
     new = Text("//a[contains(@href, '/operatingsystems/new')]")
+    delete = GenericLocatorWidget("//span[a='Delete']/a")
```

Comment:
Avoid relying on text - it may change any time and text will differ on different locale (sat6 supports multiple languages). There's `/a` tag with pretty unique attr `data-method="delete"`, i'd use it.
Also `GenericLocatorWidget` is not what we need - ideally widgetastic.patternfly's version of `Button()` widget, but it doesn't support this specific case with `/span` tag, we should probably implement our own one. Either that or use simple `Text()` for now as temporary solution.

Reply:
btw, if you have things locatable by id, you can pass just `#id` as the locator. Widgetastic detects the simple form of css locators, so `tag#id.class1.class2` is supported anywhere where a generic locator is taken.